### DISCUSSION TITLE
How to build

### DISCUSSION THREAD

**Discussion by westcomputerconsultancy** [artisan-roaster-scope/artisan#927] [Q&A] [answered]
Hello,

I've been trying to find instructions for how to build for Windows from source (I need win32 for an old laptop) but there doesn't really seem to be anything. Does anyone know if there are instructions?

Thank you,

Stephen

**Comment by roasterdave** [comment, marked as answer]
It has been over 5 years since the last 32 bit Windows build.  Many of the libraries used by Artisan no longer support 32 bit systems.  Thus Artisan supports only [64 bit OS](https://artisan-scope.org/about/).  Sorry but I can't help with a 32 bit build.

**Reply by westcomputerconsultancy** [reply]
Thanks for the reply. I understand.

I don't suppose there's either a binary for v1.1.0 (I understand the last win32 build) available anywhere? I can't see it in assets.

**Reply by roasterdave** [reply]
No, it is not available here.  Why not run Linux on that machine and use the current Artisan release?

**Reply by westcomputerconsultancy** [reply]
That's a really good and helpful idea. Thank you :) I'll try that :)

**Comment by westcomputerconsultancy** [comment]
No probs. Thanks for the reply. How about if I built 1.1.0?

**Reply by roasterdave** [reply]
The Artisan source for v1.1.0 is available in the repository.  Some caveats however.  The build environment was much different then.  You will need to create that environment by installing the necessary libraries.  There are files in the source that document the required libraries.  You may have issues with the build process or with the result.  The Artisan team can't support the old versions nor how to build them.

**Reply by westcomputerconsultancy** [reply]
Thank you ever so much for your reply. 

In the end I used your original suggestion and dual-booted Windows with Linux and installed the latest version of Artisan.
